history/src/postgres/postgres_transaction.rs

```diff
@@ -32,6 +32,81 @@ impl From<&TransactionInfo> for PostgresTransaction {
             cu_consumed: value.cu_consumed.map(|x| x as i64),
             recent_blockhash: value.recent_blockhash.clone(),
             message: value.message.clone(),
+            slot: slot as i64,
+        }
+    }
+
+    pub fn create_statement(schema: &String) -> String {
+        format!(
+            "\
+        CREATE TABLE {}.TRANSACTIONS (
+            signature CHAR(88) NOT NULL,
+            slot BIGINT, 
+            err STRING,
+            cu_requested BIGINT,
+            prioritization_fees BIGINT,
+            cu_consumed BIGINT,
+            recent_blockhash STRING NOT NULL,
+            message STRING NOT NULL,
+            PRIMARY KEY (signature)
+            CONSTRAINT fk_transactions FOREIGN KEY (slot) REFERENCES {}.BLOCKS(slot);
+          );
+        ",
+            schema, schema
+        )
+    }
+
+    pub async fn save_transactions(
+        postgres_session: &PostgresSession,
+        schema: &String,
+        transactions: &[Self],
```

Comment:
how many transactions to are we expecting here? could you document that? are the figures reasonably low to fit in one statement?